lib/index.js

```diff
@@ -22,6 +22,12 @@ var DEFAULT_DELEGATE = {
     fs.mkdirSync(outputPath);
   },
   change: function(inputPath, outputPath, relativePath) {
+    // We no-op if the platform can symlink, because we assume the output path
+    // is already linked via a prior create operation.
+    if (util.canSymlink) {
+      return;
+    }
+
     fs.unlinkSync(outputPath);
```

Comment:
This is my expectation as well, but there is technically nothing preventing that at the current moment, since the trees are based on relativePaths.